tests/ui/pages/desktop/category.py

```diff
@@ -0,0 +1,28 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
```

Comment:
Not a fan of the `EC` alias, can't we import directly from `expected_conditions` or maybe use `conditions` as a better alias?

Reply:
Well EC is nice and short...I can only think of that as a reason.

Reply:
Personally I like `expected` as it's readable in `wait.until(expected.invisibility_of_element_located('foo', 'bar'))`

Reply:
I like `expected`, can you still do that?

Reply:
Forgot this one!